stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/attachments/preview/MediaGalleryPreviewActivity.kt

```diff
@@ -1528,6 +1537,36 @@ public class MediaGalleryPreviewActivity : AppCompatActivity() {
         }
     }
 
+    /**
+     * Fetches individual image resizing options from the bundle and
+     * packs them into [StreamCdnImageResizing].
+     *
+     * @return An instance of [StreamCdnImageResizing] created from individual options fetched from
+     * the bundle packed inside the given intent.
+     */
+    private fun Intent.createStreamCdnImageResizing(): StreamCdnImageResizing {
+        val imageResizingEnabled = getBooleanExtra(KeyImageResizingEnabled, false)
+
+        val resizedWidthPercentage = getFloatExtra(KeyStreamCdnResizeImagedWidthPercentage, 0.5f)
```

Comment:
why are the defaults to halve the size?

Reply:
I have to declare _a_ number here.

The actual value missing is a case that should happen if anyone uses the code correctly using our "new instance pattern" (`MediaGalleryPreviewActivity.getIntent()`).

It could happen if someone decides to not use it, but I think in that case it could be considered a user error.

So in reality this value of `0.5f` is just something we had to declare, but it should not hold any meaning if the activity was created using the proper method.

Reply:
Yeah but any reason why it's not 1f to just not resize and use full size?